frontend/src/components/popups/WalletPage.tsx

```diff
@@ -31,7 +31,7 @@ const Walletpage = ({
             <div
```

Comment:
I suggest defining a constant for the networks array and mapping it. This way, if a new wallet is added in the future, there's no need to create another div.